GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -264,9 +279,11 @@ private void ReloadCommitInfo()
             showMessagesOfAnnotatedTagsToolStripMenuItem.Checked = AppSettings.ShowAnnotatedTagsMessages;
             showTagThisCommitDerivesFromMenuItem.Checked = AppSettings.CommitInfoShowTagThisCommitDerivesFrom;
 
+            _showAllBranches = false;
+            _showAllTags = false;
             _branches = null;
-            _annotatedTagsMessages = null;
```

Comment:
why change order
(just kidding, does not matter to me but you asked the same recently...)